food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```diff
@@ -225,10 +250,14 @@ public Route createTruckRoute(@AuthenticationPrincipal User user, @PathVariable
     @Secured("ROLE_OWNER")
     public boolean updateTruckRouteLocations(@AuthenticationPrincipal User user, @PathVariable long routeId, @RequestBody List<UpdateRouteLocationParams> data) {
         boolean good = true;
+
         for (var d : data) {
             if (d.routeLocationId != null && !routeService.userOwnsLocation(user, d.routeLocationId))
                 throw new ResponseStatusException(HttpStatus.FORBIDDEN);
-            if (!routeService.addOrUpdateLocation(routeId, d.routeLocationId, d.lat, d.lng, d.arrivalTime, d.exitTime))
+
+            LocalTime arrival = d.arrivalTime.toLocalTime();
```

Comment:
See above, I think we should take in an Instant and convert it to UTC to get the local time.

Reply:
See my comment on the first issue.